app/apollo/models/resource.schema.js

```diff
@@ -69,6 +69,13 @@ ResourceSchema.statics.getIds = async(ids)=>{
 ResourceSchema.statics.getByClusterIds = async(clusterIds)=>{
   return await this.find({ cluster_id: { $in: clusterIds } });
 };
+ResourceSchema.index({
+  cluster_id: 'text',
+  selfLink: 'text'
+}, {
+  name: 'cluster_id.selfLink',
+  default_language: 'none'
+});
```

Comment:
@harkotha2 are we able to do full-text search on searchableData and its fields ?

Reply:
@yingwang-us  - Yes, searchableData fields (`kind , name , namespace`) are will be part `SelfLink`. We should be able to search them

Reply:
As searchableData fields already part of SelfLink, I didn't added searchableData fields to resource index

<img width="1022" alt="Screenshot 2020-07-22 at 9 51 07 PM" src="https://user-images.githubusercontent.com/67068489/88201835-9b732800-cc65-11ea-83f0-d838674ea63e.png">

Reply:
@harkotha2 there are other fields that are in searchbleData but not in selfLink we would like to search as well if possible ? e.g. ```"searchableData": {
        "kind": "Namespace",
        "name": "xxx-enf",
        "apiVersion": "v1",
        "annotations[\"openshift_io_sa_scc_mcs\"]": "s0:c40,c15",
        "annotations[\"openshift_io_sa_scc_supplemental_groups\"]": "1001590000/10000",
        "annotations[\"openshift_io_sa_scc_uid_range\"]": "1001590000/10000"
      },```